### PR TITLE
Update terminology

### PR DESCRIPTION
**Pull request by rneher**
We have a number of odd terms in the code and in the interface that are largely historical accidents. This PR hopes to address some of them

## Related issues and PRs

https://github.com/neherlab/covid19_scenarios/issues/488

## Description
Search/replace to have better names and terms. 

## Impacted Areas in the application
Schema, code, interface. 

## Testing
Should not change any behavior.

### CODE REVIEW

**Comment by rneher** [comment]
Some parameter got swapped.

**Comment by noleti** [comment]
One more thing: 'severe' and 'hospitalized' are usually the same things, no? I think switching between the two terms often makes it harder to read the code, or understand the impact of the 'severe' parameter in the app, for example. Unless you later plan to differentiate those two things (e.g. severe cases not in hospital), this could also be unified?

**Comment by rneher** [comment]
good point, Nils. `hospitalized` vs `severe` is a slightly bigger change... But did you notice any problem with the rest?

File: src/locales/de/translation.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
   "Imports per day": "Importe pro Tag",
   "Infectious": "Infektiös",
   "Infectious period": "Infektiöser Zeitraum",
-  "Initial suspected cases": "Erste Verdachtsfälle",
+  "Initial number of cases": "Erste Verdachtsfälle",
```
Comment (noleti) [inline src/locales/de/translation.json:53]: Some of these translations seem a bit outdated now. For example "Anfänglich Infizierte" or so would be more appropriate now.

File: src/locales/es/translation.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
   "Imports per day": "Importaciones por día",
   "Infectious": "Infeccioso",
   "Infectious period": "Período de contagio",
-  "Initial suspected cases": "Casos sospechosos inicialmente",
+  "Initial number of cases": "Casos sospechosos inicialmente",
```
Comment (noleti) [inline src/locales/es/translation.json:51]: 'Casos inicialmente'?

File: src/locales/fr/translation.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
   "Imports per day": "Imports par jour",
   "Infectious": "Infectieux",
   "Infectious period": "Période infectieuse",
-  "Initial suspected cases": "Premiers cas suspects",
+  "Initial number of cases": "Premiers cas suspects",
```
Comment (noleti) [inline src/locales/fr/translation.json:54]: 'Premiers cas'?

File: src/locales/pl/translation.json [hunk 4]
```diff
@@ -50,7 +50,7 @@
     "Imports per day": "Dzienne przypadki zza granicy",
     "Infectious": "Zarażeni",
     "Infectious period": "Okres zarażania",
-    "Initial suspected cases": "Początkowe podejrzane przypadki",
+    "Initial number of cases": "Początkowe podejrzane przypadki",
```
Comment (noleti) [inline src/locales/pl/translation.json:53]: "Początkowo zainfekowany" (google translate of Initially infected)

File: src/locales/pt/translation.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
   "Imports per day": "Importações por dia",
   "Infectious": "Infeccioso",
   "Infectious period": "Período infeccioso",
-  "Initial suspected cases": "Casos suspeitos iniciais",
+  "Initial number of cases": "Casos suspeitos iniciais",
```
Comment (noleti) [inline src/locales/pt/translation.json:54]: Casos iniciais

**Comment by rneher** [comment]
there are a lot of untranslated strings right now. That is why we disabled the localization for now. We need to make a push to revamp these soon.

**Comment by nnoll** [comment]
Are there any major name changes outside of suspectedCasesToday -> initialNumberCases? This was sorely needed so I approve this but I can't find any other major name changes.

**Comment by rneher** [comment]
no, i only changed `suspectedCasesToday` and added changes to the print.

**Pull request merged**